main.go

```diff
@@ -364,7 +390,27 @@ func runCLIApp(c *cli.Context) (err error) {
 			err = fmt.Errorf("daemonize.Run: %w", err)
 			return
 		} else {
-			logger.Infof(SuccessfulMountMessage)
+			if isDynamicMount {
+				logger.Infof(SuccessfulMountMessage)
+			} else {
+				switch mountConfig.MetadataPrefetchMode {
+				case config.MetadataPrefetchModeDisabled:
+					logger.Infof(SuccessfulMountMessage)
+				case config.MetadataPrefetchModeSynchronous:
+					if err = callListRecursive(mountPoint); err != nil {
+						err = fmt.Errorf("daemonize.Run: metadata-prefetch failed: %w", err)
```

Comment:
Why specifically daemonize.Run - won't this error trigger in foreground mode?

Reply:
Removed 'daemonize.Run from log'.